packages/suite/src/actions/wallet/receiveActions.ts

```diff
@@ -104,6 +105,18 @@ export const showAddress =
             case 'bitcoin':
                 response = await TrezorConnect.getAddress(params);
                 break;
+            case 'solana':
+                // TODO(vl): Remove mock and replace with connect method
+                response = await Promise.resolve({
```

Comment:
The data in this response seems to be completely ignored, even though the modal will not show if there isn't a response here. I am looking into why.